lib/util/dateDiff.ts

```diff
@@ -0,0 +1,32 @@
+export const dateDiff = (createdDate: Date | undefined, nowDate: Date | undefined): string | undefined => {
+  if(nowDate !== undefined && createdDate !== undefined){
```

Comment:
Date 객체는 항상 유효한 데이터를 가지고 있기 때문에 Falsy 값이 될 수 없어 아래와 같이 작성을 권해드립니다.

```suggestion
  if(!nowDate && !createdDate){
```

Reply:
Falsy 값에 관한 내용으로는 mdn의 아래의 링크를 참고해주세요.
https://developer.mozilla.org/ko/docs/Glossary/Falsy